include/LightGBM/config.h

```diff
@@ -1105,21 +1111,24 @@ struct Config {
   #pragma region GPU Parameters
   #endif  // __NVCC__
 
+  // desc = used only with ``gpu`` device type
   // desc = OpenCL platform ID. Usually each GPU vendor exposes one OpenCL platform
   // desc = ``-1`` means the system-wide default platform
   // desc = **Note**: refer to `GPU Targets <./GPU-Targets.rst#query-opencl-devices-in-your-system>`__ for more details
   int gpu_platform_id = -1;
 
-  // desc = OpenCL device ID in the specified platform. Each GPU in the selected platform has a unique device ID
+  // desc = OpenCL device ID in the specified platform or CUDA device ID. Each GPU in the selected platform has a unique device ID
   // desc = ``-1`` means the default device in the selected platform
   // desc = **Note**: refer to `GPU Targets <./GPU-Targets.rst#query-opencl-devices-in-your-system>`__ for more details
   int gpu_device_id = -1;
 
+  // desc = used only with ``gpu`` device type
```

Comment:
Should we maybe combine this with the other line below? So just one line like this:

```text
  // desc = **Note**: can be used only in OpenCL implementation (``device_type="gpu"``), in CUDA implementation only double precision is currently supported
```

I think that might be a bit clearer. I think some users are confused about language like "OpenCL implementation", and by the difference between the two different forms of GPU support here.

Reply:
Sure, very clean suggestions! Just committed them both it in 50ec8e690a8aee2cc1b049bae45ff7ee244be040.

Reply:
thanks!